common-compose.yml

```diff
@@ -0,0 +1,70 @@
+wordpress:
```

Comment:
i'm working on merging these files and getting rid of common-compose but I'm running to trouble because I can't extend a service that has links:
```
ERROR: Cannot extend service 'nginx' in /home/jason/work/joyent/joyent_repos/triton-wordpress/docker-compose.yml: services with 'links' cannot be extended```

now that I see this, it was the main reason for going the common-compose route. I do have the links defined in local-compose, but we will still need the links when running on triton, right? Or do we use consul's DNS service or something?